internal/compiler/widgets/common/menu-base.slint

```diff
@@ -139,6 +139,7 @@ export component MenuItemBase {
                     Image {
```

Comment:
I wonder if the image shouldn't be placed bofore the check mark.

Reply:
Let me know if you'd like me to change this.